Automattic/woocommerce-payments
# Transaction Details – display headline currency in store's currency rather than shopper's currency

Part of #7124

> **Warning**
> This PR is **NOT** behind a feature flag

#### Changes proposed in this Pull Request

This PR displays the Transaction Details headline payment amount in the store's currency when the store/charge currency differs (for multi-currency enabled stores).

[See design](https://www.figma.com/file/RiEQmaKRI7u9USAI3lyZz0VX/Dispute-Flows?node-id=5482%3A443300&mode=dev). 

> [!Note]
> This PR does not change the display currency of the new dispute summary amount – keeping feature-flag and non-feature-flag changes separate. See PR #7287.

#### Before/After

The screenshots below are for a transaction where:
* the store has multi-currency enabled
* the store is an Australian account with **AUD** set as the store's primary deposit currency
* the purchase was made with **USD** selected at checkout

**Before** – USD (charge currency) shows as headline amount

![image](https://github.com/Automattic/woocommerce-payments/assets/3147296/fe3b57e9-8488-4d38-bf93-1f90e50a92d2)

**After** – AUD (store currency) shows as headline amount
![image](https://github.com/Automattic/woocommerce-payments/assets/3147296/bdfee5fe-1459-449f-a364-8f648d21e786)

#### Testing instructions





1. Enable **Payments → Settings → Multi-currency**.
3. Visit **Payments → Overview** and note your store's deposit currency or currencies, visible as balance tabs.
4. Enable a non-store currency in **WooCommerce → Settings → Multi-currency**. Enable the currency switcher if using the Storefront theme (otherwise, show the front-end currency switcher via the block editor or another method). 
6. Create a transaction in (one of) the store's currency. The transaction details screen should show the headline payment amount in the store's currency. (example: a EUR charge currency and a EUR store currency)
    ![image](https://github.com/Automattic/woocommerce-payments/assets/3147296/03ccb626-a733-40ab-8ed1-8fc08f7adec2)
7. Create a transaction in a non-store currency using the currency switcher.
8. The transaction details screen should show the headline payment amount in the store's currency with the charge currency rendered below. (example: a JPY charge currency and a NOK store currency)
    ![image](https://github.com/Automattic/woocommerce-payments/assets/3147296/84157741-d193-4236-a199-ce9a67b0c3f4)

-------------------


**Post merge**
